site/docs/skylark/language.md

```diff
@@ -37,142 +29,97 @@ def fizz_buzz(n):
 
 ## Differences with Python
 
-In addition to the mutability restrictions, there are also differences with
-Python:
-
-* Global variables cannot be reassigned.
+* Global variables are immutable.
 
-* `for` statements are not allowed at the top-level; factor them into functions
-  instead. In BUILD files, you may use list comprehensions.
+* `for` statements are not allowed at the top-level. Use them within functions instead. In BUILD files, you may use list comprehensions.
 
-* `if` statements are not allowed at the top-level. However, `if` expressions
-  can be used: `first = data[0] if len(data) > 0 else None`.
+* `if` statements are not allowed at the top-level. However, `if` expressions can be used: `first = data[0] if len(data) > 0 else None`.
 
-* Dictionaries have a deterministic order of iteration.
+* Deterministic order for iterating through Dictionaries.
 
 * Recursion is not allowed.
 
-* Int type is limited to 32-bit signed integers (an overflow will throw an
-  error).
+* Int type is limited to 32-bit signed integers. Overflows will throw an error.
 
-* Modifying a collection during iteration is an error. You can avoid the error
-  by iterating over a copy of the collection, e.g.
-  `for x in list(my_list): ...`. You can still modify its deep contents
-  regardless.
+* Modifying a collection during iteration is an error.
 
-* The comparison operators (`<`, `<=`, `>=`, `>`) are not defined across
-  different types of values, e.g., you can't compare `5 < 'foo'` (however you
-  still can compare them using `==` or `!=`). This is a difference with Python
-  2, but consistent with Python 3. Note that this means you are unable to sort
-  lists that contain mixed types of values.
+* Except for equality tests, comparison operators `<`, `<=`, `>=`, `>`, etc. are not defined across value types. In short: `5 < 'foo'` will throw an error and `5 == "5"`
+will return false.
 
-* Tuple syntax is more restrictive. You may use a trailing comma only when the
-  tuple is between parentheses, e.g. write `(1,)` instead of `1,`.
+* In tuples, a trailing comma is valid only when the tuple is between parentheses, e.g. write `(1,)` instead of `1,`.
 
-* Dictionary literals cannot have duplicated keys. For example, this is an
-  error: `{"a": 4, "b": 7, "a": 1}`.
+* Dictionary literals cannot have duplicated keys. For example, this is an error: `{"a": 4, "b": 7, "a": 1}`.
 
-* Variable of a comprehension may not be used after the comprehension. This is
-  stricter than Python 2 and Python 3, which have different behavior (shadowing
-  vs reassignment).
+* The variable used in a comprehension may not be used after the comprehension. This is stricter than Python 2 and Python 3, which have different behavior (shadowing vs reassignment).
 
-* Strings are represented with double-quotes (e.g. when you
-  call [repr](lib/globals.html#repr)).
+* Strings are represented with double-quotes (e.g. when you call [repr](lib/globals.html#repr)).
```

Comment:
"Strings are conventionally represented with"

This is to help avoid implying that the literal syntax `s = 'abc'` is invalid.

Reply:
thank you! I updated it.